GitUI/CommandsDialogs/RevisionFileTreeControl.Designer.cs

```diff
@@ -244,82 +246,90 @@ private void InitializeComponent()
             this.openWithToolStripMenuItem.Image = global::GitUI.Properties.Images.EditFile;
             this.openWithToolStripMenuItem.Name = "openWithToolStripMenuItem";
             this.openWithToolStripMenuItem.ShortcutKeys = ((System.Windows.Forms.Keys)((System.Windows.Forms.Keys.Control | System.Windows.Forms.Keys.O)));
-            this.openWithToolStripMenuItem.Size = new System.Drawing.Size(325, 22);
+            this.openWithToolStripMenuItem.Size = new System.Drawing.Size(346, 22);
             this.openWithToolStripMenuItem.Text = "Open working directory file with...";
             this.openWithToolStripMenuItem.Click += new System.EventHandler(this.openWithToolStripMenuItem_Click);
             // 
+            // toggleExecutePermissionsToolStripMenuItem
+            // 
+            this.toggleExecutePermissionsToolStripMenuItem.Image = global::GitUI.Properties.Images.EditFile;
+            this.toggleExecutePermissionsToolStripMenuItem.Name = "toggleExecutePermissionsToolStripMenuItem";
+            this.toggleExecutePermissionsToolStripMenuItem.Size = new System.Drawing.Size(346, 22);
+            this.toggleExecutePermissionsToolStripMenuItem.Text = "Toggle execute permission in working directory file...";
```

Comment:
Sorry for being pedantic about the wording, but in English we usually say
> Toggle execute permission on a file.

("on" instead of "in"). Since this text is meant to be less specific than talking about a specific single file (visualized by the ellipsis), it is OK to phrase it like
```
"Toggle execute permission on working directory file...";
```
(just a 1 char diff - oh well ;)

Reply:
Fixed :)